### DISCUSSION TITLE
A broken link from FastAPI docs leading to a non-existent Pydantic page

### DISCUSSION THREAD

**Discussion by ladmol** [fastapi/fastapi#10508] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
No code required
```

### Description

Between two tips of [this paragraph](https://fastapi.tiangolo.com/advanced/settings/#reading-a-env-file) there is a link leading to the Pydantic documentation. Apparently Pydantic changed the link, so now the FastAPI documentation leads to a non-existent Pydantic page.
We need to change the link to [this one](https://docs.pydantic.dev/latest/concepts/pydantic_settings/#dotenv-env-support)

### Operating System

Linux, Windows, macOS, Other

### Operating System Details

It doesn't depend on the system

### FastAPI Version

.

### Pydantic Version

.

### Python Version

.

**Comment by White-Mask** [comment]
Hello @ladmol , indeed, the link is outdated. You need to create an issue for this problem.

**Reply by ladmol** [reply]
Hello, @White-Mask. Thanks! I'll wait for @tiangolo's permission to create an issue.

**Comment by White-Mask** [comment, marked as answer]
@ladmol , could you please check if the Pydantic links are working correctly now? I believe it's fine from my end. I await your confirmation to request the merge. Regards
pr: https://github.com/tiangolo/fastapi/pull/10511
Docs preview for commit https://github.com/tiangolo/fastapi/commit/1c451b0cd05862c9218da60548ea156bc4cbfcb6 at: https://ddf09f5a.fastapitiangolo.pages.dev/

**Reply by ladmol** [reply]
@White-Mask, yep, everything works fine!
